docs/_sass/_main.scss

```diff
@@ -456,12 +456,26 @@ button {
     h3.title {
         padding: 0;
         margin: 0;
+
+        &.with-margin {
+            margin: 0 0 8px 0;
```

Comment:
I think this might look better with the bottom margin being 4px if you don't mind changing that out quickly, thanks!

Reply:
Sure, updated!